load_id : 1   # Choose Load '1','2','3', or '0' for no load

input:
  frequency: 200

  type: 2     # 0: 'step' | 1: 'chirp' | 2: 'flight'
  centre_point: 0

  # Params for random step input
  step:
    frequency: 5
    variance: 0.1

  # Params for chirp input
  chirp:
    amplitude: 0.5
    frequency1: 10
    frequency2: 40
    period: 2

  flight:
    rec_frequency: 200
    file: "23-02-08--21-26-11_ID5.csv"